src/client/datascience/variablesView/variableView.ts

```diff
@@ -82,6 +82,7 @@ export class VariableView extends WebviewViewHost<IVariableViewPanelMapping> imp
         this.dataViewerChecker = new DataViewerChecker(configuration, appShell);
     }
 
+    @captureTelemetry(Telemetry.NativeVariableViewLoaded)
```

Comment:
This seemed extra useful as we are not using retain context on the webview. Means that it gets loaded fairly often. Here to make sure that this is not taking too long.